torch/onnx/verification.py

```diff
@@ -136,136 +72,217 @@ def _run_ort(ort_sess, inputs):
     model: Union[torch.nn.Module, torch.jit.ScriptModule],
-    input=None,
-    batch_size=2,
-    rtol=0.001,
-    atol=1e-7,
+    input_args,
+    input_kwargs=None,
     do_constant_folding=True,
     dynamic_axes=None,
-    test_with_inputs=None,
     input_names=None,
     output_names=None,
-    fixed_batch_size=False,
-    dict_check=True,
     training=None,
+    opset_version=None,
+    keep_initializers_as_inputs=True,
+    verbose=False,
+    fixed_batch_size=False,
+    use_external_data=False,
+    additional_test_inputs=None,
     remained_onnx_input_idx=None,
     flatten=True,
-    verbose=False,
     ort_providers=_ORT_PROVIDERS,
-    ort_optim_on=True,
-    use_external_data=False,
-    opset_version: Optional[int] = None,
-    keep_initializers_as_inputs: bool = True,
+    rtol=0.001,
+    atol=1e-7,
+    **kwargs,
 ):
-    """Verify model export to ONNX with ONNXRuntime."""
+    """Verify model export to ONNX with ONNXRuntime.
+
+    Args:
+        model (torch.nn.Module or torch.jit.ScriptModule): See :func:`torch.onnx.export`.
+        input_args (tuple): See :func:`torch.onnx.export`.
+        input_kwargs (dict): See :func:`torch.onnx.export`.
+        do_constant_folding (bool, optional): See :func:`torch.onnx.export`.
+        dynamic_axes (dict, optional): See :func:`torch.onnx.export`.
+        input_names (list, optional): See :func:`torch.onnx.export`.
+        output_names (list, optional): See :func:`torch.onnx.export`.
+        training (bool, optional): See :func:`torch.onnx.export`.
+        opset_version (int, optional): See :func:`torch.onnx.export`.
+        keep_initializers_as_inputs (bool, optional): See :func:`torch.onnx.export`.
+        verbose (bool, optional): See :func:`torch.onnx.export`.
+        fixed_batch_size (bool, optional): Legacy argument, used only by rnn test cases.
+        use_external_data (bool, optional): Explicitly specify whether to export the
+            model with external data.
+        additional_test_inputs (list, optional): List of tuples. Each tuple is a set of
+            input arguments to test. Currently only *args are supported.
+        remained_onnx_input_idx (list, optional): If set, only the specified inputs will
+            be passed to ONNX model. This is used when there are unused inputs in original
+            PyTorch model. ONNX model will remove unused inputs automatically.
+        flatten (bool, optional): Default True. If True, unpack nested list/tuple/dict
+            inputs into a flattened list of Tensors for ONNX. Set this to False if nested
+            structures are to be preserved for ONNX, which is usually the case with
+            exporting ScriptModules.
+        ort_providers (sequence, optional): ONNXRuntime providers to use.
+        rtol (float, optional): relative tolerance in comparison between ONNX and PyTorch outputs.
+        atol (float, optional): absolute tolerance in comparison between ONNX and PyTorch outputs.
+    """
```

Comment:
Does the function raise any exceptions depending on the comparison result? If so I would add a "Raises" section in the docstring.